### PR TITLE
build: upgrade node and packages

### PR DESCRIPTION
**Pull request by RonitKissis**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔/✖                                                                       |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,37 +46,37 @@
     }
   },
   "dependencies": {
-    "@godaddy/terminus": "^4.11.2",
+    "@godaddy/terminus": "^4.12.1",
     "@map-colonies/error-express-handler": "^2.1.0",
     "@map-colonies/error-types": "^1.1.0",
```
Comment (asafMasa) [inline package.json:51]: [v1.1.5](https://github.com/MapColonies/error-types/releases/tag/v1.1.5)

File: package.json [hunk 2]
```diff
@@ -46,37 +46,37 @@
     }
   },
   "dependencies": {
-    "@godaddy/terminus": "^4.11.2",
+    "@godaddy/terminus": "^4.12.1",
     "@map-colonies/error-express-handler": "^2.1.0",
```
Comment (asafMasa) [inline package.json:50]: [v2.1.0](https://github.com/MapColonies/error-express-handler/releases/tag/v2.1.0)

File: package.json [hunk 3]
```diff
@@ -46,37 +46,37 @@
...
     "@map-colonies/error-types": "^1.1.0",
     "@map-colonies/express-access-log-middleware": "^2.0.1",
     "@map-colonies/js-logger": "^1.0.1",
     "@map-colonies/mc-model-types": "^14.1.2",
```
Comment (asafMasa) [inline package.json:54]: 15.2.0

**Review by asafMasa — CHANGES_REQUESTED** [review]
please try to upgrade thos version also

File: package.json (hunk 3 above)
Comment (RonitKissis) [reply package.json:54]: cant upgrade, there are breaking changes.

**Review by asafMasa — APPROVED** [review]

**Pull request merged**